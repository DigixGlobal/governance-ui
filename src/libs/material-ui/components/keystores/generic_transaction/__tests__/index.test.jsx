import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import GenericTransactionForm from '../index';

jest.mock('react-redux', () => ({
  connect: () => component => component,
}));

describe('<GenericTransactionForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <GenericTransactionForm />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
