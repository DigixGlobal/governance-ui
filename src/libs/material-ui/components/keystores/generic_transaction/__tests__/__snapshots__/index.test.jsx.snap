// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GenericTransactionForm /> renders correctly 1`] = `
<TransactionModal
  data={Object {}}
  disableAdvanced={true}
  handleTransaction={[Function]}
  header="Generic Transaction Signer"
  renderForm={[Function]}
  trigger={
    <Button
      as="button"
      basic={true}
      content="Sign"
      icon="pencil"
      onClick={[Function]}
    />
  }
/>
`;
